spec/system/provider_interface/provider_applications_sort_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.feature 'Providers should be able to sort applications' do
+  include CourseOptionHelpers
+  include DfESignInHelpers
+
+  # rubocop:disable RSpec/ExpectActual
```

Comment:
Maybe scope this tightly around the assertions that need it, ie `then_i_should_see_the_applications_in_descending_date_order`?